test/parallel/test-dgram-send-callback-buffer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
+var fs = require('fs');
+var dgram = require('dgram');
+var callbacks = 0;
+var client, timer, buf;
+
+client = dgram.createSocket('udp4');
+
+buf = new Buffer(256);
+
+client.send(buf, common.PORT, '127.0.0.1', function(err, bytes) {
+  assert.equal(bytes, buf.length);
+  clearTimeout(timer);
+  client.close();
+});
+
+timer = setTimeout(function() {
+  throw new Error('Timeout');
+}, 200);
```

Comment:
Why do we need this? We have few boxes which run very slow compared to the others. So, this 200 might not work well with all of them.